### PR TITLE
Adding wildcard support for CsOnlineLisWirelessAccessPoint

### PR DESCRIPTION
**Pull request by kaishuipinggai**
Adding wildcard support for CsOnlineLisWirelessAccessPoint

### CODE REVIEW

File: skype/skype-ps/skype/Get-CsOnlineLisWirelessAccessPoint.md [hunk 1]
```diff
@@ -24,6 +24,10 @@ Get-CsOnlineLisWirelessAccessPoint [[-TenantId] <Guid>] [[-BSSID] <String>] [-Is
 ## DESCRIPTION
 Enhanced 9-1-1 allows an emergency operator to identify the location of a caller without having to ask the caller for that information. In the case where a caller is calling from a Voice over Internet Protocol (VoIP) connection, that information must be extracted based on various connection factors. The VoIP administrator must configure a location map (called a wiremap) that will determine a caller's location. This cmdlet retrieves information on associations between physical locations and the WAP through which the client is connected.
 
+The BSSID (Basic Service Set Identifiers) is used to describe sections of a wireless local area network. It is the MAC of the 802.11 side of the access point. The BSSID in this command is also supporting the wildcard format which is covering all BSSIDs in the range which are sharing the same description and Location ID. The wildcard can be one '*' on either the last one or two character(s).
```
Comment (get-itips) [inline skype/skype-ps/skype/Get-CsOnlineLisWirelessAccessPoint.md:27]:
```suggestion
The BSSID (Basic Service Set Identifiers) is used to describe sections of a wireless local area network. It is the MAC of the 802.11 side of the access point. The BSSID parameter in this command is also supporting the wildcard format which is covering all BSSIDs in the range which are sharing the same description and Location ID. The wildcard can be one '*' on either the last one or two character(s).
```

Comment (kaishuipinggai) [reply skype/skype-ps/skype/Get-CsOnlineLisWirelessAccessPoint.md]: Thanks @get-itips I have updated the change

**Comment by kaishuipinggai** [comment]
@dariomws In Get- command example 4, when we already have a record "CB-FF-8D-44-3C-4*", this means the wildcard BSSID is covering all the records in its range. When the user types in "CB-FF-8D-44-3C-44", apparently we don't have "CB-FF-8D-44-3C-44" but we have "CB-FF-8D-44-3C-4*" instead. So the user will see "CB-FF-8D-44-3C-4*" instead of the original BSSID they typed in.

**Comment by dariomws** [comment]
> @dariomws In Get- command example 4, when we already have a record "CB-FF-8D-44-3C-4*", this means the wildcard BSSID is covering all the records in its range. When the user types in "CB-FF-8D-44-3C-44", apparently we don't have "CB-FF-8D-44-3C-44" but we have "CB-FF-8D-44-3C-4*" instead. So the user will see "CB-FF-8D-44-3C-4*" instead of the original BSSID they typed in.

Thanks @kaishuipinggai! It´s clear now. Is this not the same as example 2?

**Comment by kaishuipinggai** [comment]
@dariomws It's not the same as example 2. I will update the doc to make it more clear.

**Comment by dariomws** [comment]
@kaishuipinggai I think I understand that it's not the same, but it could be confusing.

In Example 2, you'll get 0B-23-CD-16-BB-CC.

In Example 4, you'll get CB-FF-8D-44-3C-4*. But it´s the same if you run "Get-CsOnlineLisWirelessAccessPoint -BSSID CB-FF-8D-44-3C-48" or "Get-CsOnlineLisWirelessAccessPoint -BSSID CB-FF-8D-44-3C-4*", right?

**Comment by kaishuipinggai** [comment]
@dariomws 

> In Example 2, you'll get 0B-23-CD-16-BB-CC.

> In Example 4, you'll get CB-FF-8D-44-3C-4*. But it´s the same if you run "Get-CsOnlineLisWirelessAccessPoint -BSSID CB-FF-8D-44-3C-48" or "Get-CsOnlineLisWirelessAccessPoint -BSSID CB-FF-8D-44-3C-4*", right?

That is correct.

Also do you know how to add our PM as a reviewer? He created some review comments but they are in "Pending" state.

**Comment by dariomws** [comment]
Ok, it's clear for me now. I'm not sure how you can provide a better explanation so I think you have done a good job. Thanks!

I don't know who the PM is, you can tag him if you know her/his username or maybe @yogkumgit can help us to add him as a reviewer.

**Comment by kaishuipinggai** [comment]
@roykuntz is our team's PM.

**Comment by yogkumgit** [comment]
@roykuntz Please review this PR and help us to resolve it. Thanks.

**Comment by kaishuipinggai** [comment]
Can I get this change merged into master? Thanks! @get-itips @dariomws

**Comment by get-itips** [comment]
> Can I get this change merged into master? Thanks! @get-itips @dariomws
We will need @chrisda to kindly to that.

**Comment by kaishuipinggai** [comment]
@chrisda Can I have this review merged into master? Thanks a lot!

**Pull request merged**